Tests/FaunaDBTests/ClientTests.swift

```diff
@@ -1,5 +1,6 @@
 import XCTest
-import FaunaDB
+
+@testable import FaunaDB
```

Comment:
I just changed it in order to be able to instantiate an `QueryV` manually to test echo of that type and see if we're not wrapping the internal lambda wrongly